nano/templates/freezer.tmpl

```diff
@@ -39,7 +39,7 @@
 			{{:helper.link('-', null, {'temp' : -100}, (data.targetGasTemperature > data.minGasTemperature) ? null : 'disabled')}}
 			{{:helper.link('-', null, {'temp' : -10}, (data.targetGasTemperature > data.minGasTemperature) ? null : 'disabled')}}
 			{{:helper.link('-', null, {'temp' : -1}, (data.targetGasTemperature > data.minGasTemperature) ? null : 'disabled')}}				
-			<div style="float: left; width: 80px; text-align: center;">&nbsp;{{:data.targetGasTemperature}} K&nbsp;</div>
+			<div style="float: left; width: 110px; text-align: center;">&nbsp;{{:data.targetGasTemperature}} K&nbsp; ({{:data.targetGasTemperatureCelsius}} &deg;C)</div>
```

Comment:
The `&nbsp;` between `center;">` and `{{:data` doesn't really do anything and shifts the text center.